src/core/common/permissions/rules.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  isLTESiteModerator,
+  isOrgModerator,
+  isSiteModerator,
+  PermissionsActionRuleTest,
+} from "./types";
+// admins can do whatever they want to anyone except other admins
+const adminsArePowerful: PermissionsActionRuleTest = ({ viewer, user }) => ({
+  applies: viewer.role === "ADMIN" && user.role !== "ADMIN",
+  reason: "Admins may change any non admin's role or scopes",
+});
+
+// org mods can promote anyone < site mod to be an unscoped member
```

Comment:
This is a lot of comments, and was honestly to help me think, but I feel like this should be documented somewhere. Maybe I'll create a Slab doc and put a link to it here